src/backend/api/Fusion.Resources.Domain/Queries/GetContractPersonnel.cs

```diff
@@ -34,10 +34,10 @@ public ODataQueryParams Query
 
                 if (value != null)
                 {
-                    if (value.ShoudExpand("requests"))
+                    if (ODataParamsExtensions.ShouldExpand(value, "requests"))
```

Comment:
Extension does not work here?

Reply:
Good catch, thought I had fixed that all places. VS rename fucked that up everywhere it was used 😢